jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/OdePsmDataCreatorHelper.java

```diff
@@ -0,0 +1,59 @@
+package us.dot.its.jpo.ode.coder;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import us.dot.its.jpo.ode.context.AppContext;
+import us.dot.its.jpo.ode.model.OdePsmData;
+import us.dot.its.jpo.ode.model.OdePsmMetadata;
+import us.dot.its.jpo.ode.model.OdePsmPayload;
+import us.dot.its.jpo.ode.model.ReceivedMessageDetails;
+import us.dot.its.jpo.ode.model.RxSource;
+import us.dot.its.jpo.ode.plugin.j2735.builders.PSMBuilder;
+import us.dot.its.jpo.ode.util.JsonUtils;
+import us.dot.its.jpo.ode.util.XmlUtils;
+import us.dot.its.jpo.ode.util.XmlUtils.XmlUtilsException;
+
+public class OdePsmDataCreatorHelper {
+
+	public OdePsmDataCreatorHelper() {
+	}
```

Comment:
Is this constructor necessary?

Reply:
Honestly, I am not too sure. I just followed the convention with all the other Ode{dataflow name}DataCreatorHelper.java files.